apps/prairielearn/src/pages/administratorInstitutions/administratorInstitutions.sql

```diff
@@ -21,3 +21,19 @@ FROM
   institutions AS i
 ORDER BY
   i.id ASC;
+
+-- BLOCK add_institution
```

Comment:
Nit: typically we name insertion queries `insert_*`, so this would be `insert_institution`. The more general pattern here is that we often try to use a SQL verb (`select`, `insert`, `update`, `delete`) in query names when it makes sense to.

Reply:
That makes sense! Updated the query to `insert_institution` in commit [ed4f9f4](https://github.com/PrairieLearn/PrairieLearn/pull/9045/commits/ed4f9f44057278fce12a5e2de1948446144ae6c6).